### PR TITLE
Use language names

### PR DESCRIPTION
**Pull request by stefanheule**

### CODE REVIEW

File: polyglot_piranha.pyi [hunk 1]
```diff
@@ -13,7 +13,7 @@ from __future__ import annotations
 from typing import List, Optional, Literal
 
 # Languages that Piranha supports (see ./src/models/language.rs)
-PiranhaLanguage = Literal["java", "kt", "kotlin", "go", "py", "swift", "ts", "tsx", "thrift", "strings", "scm", "scala", "rb", "yaml", "yml"]
+PiranhaLanguage = Literal["java", "kt", "kotlin", "go", "python", "swift", "typescript", "tsx", "thrift", "strings", "scm", "scala", "ruby", "yaml", "yml"]
```
Comment (yuxincs) [inline polyglot_piranha.pyi:16]: Quick question: is this a safe change? 

Looking at the names here it feels that we are probably also using this as file extensions? 

Having `kt` and `kotlin` here also seems to agree with that theory. Should we include `py` and `ts` here as well?

File: src/models/default_configs.rs [hunk 2]
```diff
@@ -25,15 +25,15 @@ pub const JAVA_CS: &str = "java_cs";
 pub const KOTLIN: &str = "kt";
 pub const KOTLIN_ALIAS: &str = "kotlin";
```
Comment (yuxincs) [inline src/models/default_configs.rs:26]: Continuing the discussion above, here we have an alias for kotlin, can we look at the uses of this and see if we need `PYTHON_ALIAS` and `TYPESCRIPT_ALIAS`?

**Review by yuxincs — APPROVED** [review]
Two questions due to my unfamiliarity with the codebase :)

But stamping in case you think it is actually not a problem

**Comment by stefanheule** [comment]
Discussed offline.

File: polyglot_piranha.pyi (hunk 1 above)
Comment (danieltrt) [reply polyglot_piranha.pyi:16]: a bit of the blue, but I added the this alias because it would otherwise break something. I don't remember exactly what

**Pull request merged**